### ISSUE TITLE
'About Us' section appear out of Simulator screen

### ISSUE THREAD

**Issue by yadav-rahul** [BuildmLearn/BuildmLearn-Toolkit-Android#379]
When an app is running on `Simulator`, Dialog box goes out of the simulator scrren when an user clicks on **About Us** section (present in ToolBar).

![rsz_1device-2017-01-04-025404](https://cloud.githubusercontent.com/assets/15157620/21623667/89dc7ad6-d229-11e6-837a-56ed16203a66.png)

@croozeus @opticod What do you feel about it? Shall I make PR for this?

**Comment by anuradhagupta345** [comment]
@yadav-rahul What is your progress in this issue?

**Comment by opticod** [comment]
@yadav-rahul, yes the dialog box should look like as if it is inside the simulator.

**Comment by yadav-rahul** [comment]
@opticod Thanks for your review. I will make PR for this soon :)
